tests/bug-5464-verdict-06/test.yaml

```diff
@@ -9,6 +9,22 @@ args:
 
 checks:
   - filter:
+      min-version: 8
+      count: 25
```

Comment:
The rule here is
```
alert http any any -> any any (sid: 1;)
```
Should it match on the stream end packets?

@catenacyber are there cases where only at the end of the stream, with the pseudo packets we detect the app protocol? I guess we could, if data stays un-ack'd, then stream terminates, FFR will force handling of un-ACK'd data, detect protocol.

Reply:
I also guess we could indeed...

Reply:
https://github.com/OISF/suricata-verify/pull/2148/commits/45b19c663ad0f8aa6fe549513d854efd9dc11ac2

Reply:
It matched in master on sid 2, in https://github.com/OISF/suricata/pull/12169 it won't match.